app/views/UploadFileView.scala.html

```diff
@@ -59,7 +59,7 @@ <h1 class="govuk-label-wrapper">
 
         </div>
 
-        <div id="processing" aria-live="polite" role="status" class="govuk-!-margin-bottom-5 govuk-!-margin-top-5">
```

Comment:
@koladeadewuyi It needs the `aria-live="polite"` left in. I did try removing it in an earlier attempt to fix the issue for MDR but it didn't work so it needs to remain.

Reply:
Fixed now. Thanks @BriWak